src/ovs/ovs_control.cpp

```diff
@@ -235,23 +236,22 @@ OVS_Control::packet_out(const char *bridge, const char *options)
     free(po.ofpacts);
 }
 
-void
-OVS_Control::dump_flows(const char *bridge, const char *opt)
+int
+OVS_Control::dump_flows(const char *bridge, const char *flow, bool show_stats)
```

Comment:
could show_stats be a default argument? 

https://en.cppreference.com/w/cpp/language/default_arguments

Reply:
do we want to make show_stats default to true using the default arguments method?